### ISSUE TITLE
Geocoder::ConfigurationError when attempting Google Places search

### ISSUE THREAD

**Issue by realmadrid2727** [alexreisner/geocoder#1149]
I'm seeing the following when using the :google_places_search lookup option as specified in the README. I'm on the latest version (1.4.2)

```
[7] pry(main)> Geocoder.search([loc.latitude, loc.longitude], lookup: :google_places_search)
Geocoder::ConfigurationError: Please specify a valid lookup for Geocoder (:google_places_search is not one of: :location_iq, :dstk, :esri, :google, :google_premier, :google_places_details, :bing, :geocoder_ca, :geocoder_us, :yandex, :nominatim, :mapbox, :mapquest, :mapzen, :opencagedata, :ovi, :pelias, :here, :baidu, :geocodio, :smarty_streets, :okf, :postcode_anywhere_uk, :geoportail_lu, :ban_data_gouv_fr, :test, :latlon, :baidu_ip, :freegeoip, :geoip2, :maxmind, :maxmind_local, :telize, :pointpin, :maxmind_geoip2, :ipinfo_io, :ipapi_com).
```

**Comment by alexreisner** [comment]
Sorry about that. Google Places Search will not be included until version 1.4.3. Until then you should install from the current `HEAD`.

**Comment by realmadrid2727** [comment]
Thanks, Alex. And awesome library!
